pytuya/__init__.py

```diff
@@ -240,14 +231,14 @@ def status(self):
         log.debug('result=%r', result)
         #result = data[data.find('{'):data.rfind('}')+1]  # naive marker search, hope neither { nor } occur in header/footer
         #print('result %r' % result)
-        if result.startswith(b'{'):
+        if result.startswith('{'):
```

Comment:
as per early comments, this won't work with py3

Reply:
I am a little surprised, as I used Python 3.4.3 during testing. And I changed it just because Python complained about str to bytes comparison. Maybe I used wrong types when calling the function...

Reply:
I've seen odd problems with older versions of python 3 and bytes. This is probably one of them, sockets return bytes, not (unicode) strings. I'm actively targeting Python 2.7 and 3.6+ (because of odd behaviors of older versions), I'll accept PRs for older versions to be supported but I'm not likely to work on that.

Cool thing is, your unittests helped prove this, they rock :)